channels_postgres/migrations/0004_notify_ensure_payload_not_too_big.py

```diff
@@ -0,0 +1,50 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [('channels_postgres', '0003_notify_smaller_payload')]
+
+    # For messages than are smaller than 7168 bytes, we can send the whole message in the payload
+    # Otherwise, we send only the message id
+
+    # Postgres' NOTIFY messages are limited to 8000 bytes, so we can't always send the whole message
+    # in the payload.
+    setup_database_sql = """
+        CREATE OR REPLACE FUNCTION channels_postgres_notify()
+        RETURNS trigger AS $$
+        DECLARE
+            payload text;
+            encoded_message text;
+            epoch text;
+        BEGIN
+            encoded_message := encode(NEW.message, 'base64');
+            epoch := extract(epoch from NEW.expire)::text;
+            IF octet_length(NEW.id::text) + octet_length(NEW.channel::text) + octet_length(encoded_message) + octet_length(epoch) + 3 <= 8000 THEN
```

Comment:
is the `+3` (in `octet_length(epoch) + 3`) meant to account for the colons that separate parts of the message i.e `id<:>channel<:>encoded_message<:>epoch`.

If yes, I would add a comment about it in the code.